sematic/examples/testing_pipeline/pipeline.py

```diff
@@ -48,6 +52,7 @@ def add_with_ray(a: float, b: float, cluster_address: str) -> float:
 def add_ray_task(x, y):
     # create new logger due to this:
     # https://stackoverflow.com/a/55286452/2540669
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Doesn't this override the user-configured logging level for the rest of the funcs?
Should that user configuration apply to this func as well?
Is this testing code?